src/templates/policies.bicep

```diff
@@ -81,7 +81,6 @@ var loadPolicyDefinitions = {
     loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Deny-PostgreSql-http.json')
     loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Deny-Private-DNS-Zones.json')
     loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Deny-PublicEndpoint-MariaDB.json')
-    loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Deny-PublicIP.json')
```

Comment:
do we want to take it out, surely leave it deploying with deprecated for now right? Otherwise in downstream repos it will just be a flat removal and no notice of deprecation?

Reply:
Good point, I'll re-introduce